source/includes/_evidence.md

```diff
@@ -0,0 +1,354 @@
+
+Multiple steps need to be taken to submit an evidence  document and it is not possible to combine these steps within one request. Similar to that in the UI, each evidence submission has to indicate file type, a valid URL pointing to the evidence, and notes. An additional step can be taken to change the setting for sharing the evidence document to investors. This setting is not applicable when submitting evidence as a hyperlink.
+
+## Upload Hyperlink as Evidence
+
+In order to upload a hyperlink as evidence, total of 3 requests should be send. First one is to indicate file type, second one to post the hyperlink URL, and final one to share evidence notes.
+
+In this example, let us share a public URL as evidence for the indicator PO1 in Management > Policies.
+
+<aside class="warning">
+Mind that PO1 has a main question which can be answered as "Yes" or "No". We are submitting the evidence under the main answer "Yes", and not for any of the checkbox variables that are presented under "Yes".
+</aside>
+
+<aside class="warning">
+The number at the end of the endpoint indicates the evidence number. This number is incrementally increased as you add more evidence to an answer (variable). In the following examples the first evidence item will be hyperlink, and second one will be file. Even if we removed the hyperlink evidence the endpoint and the item number would remain the same for the evidence file for our reference.
+</aside>
+
+## POST values/{variable} - Indicate File Type
+Indicate filetype as hyperlink by sending `H` as the value.
+
+```shell
+curl https://api.gresb.com/api/v0/entities/{entity_id}/responses/{response_id}/values/PO_1_A1_EVD_TYPE_1 \
+  -H "Authorization: Bearer $ACCESS_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d @- <<JSON
+```
+
+> Request:
+
+```json
+{
+	"lock_version": "1",
+	"value": "H"
+}
+```
+
+> Response:
+
+```json
+{
+    "id": 12345678,
+    "analysis_id": 12345,
+    "variable": "PO_1_A1_EVD_TYPE_1",
+    "value": "H",
+    "created_at": "2022-11-29T12:58:42.895Z",
+    "updated_at": "2022-12-08T14:08:00.885Z",
+    "validator_id": null,
+    "validation_status": null,
+    "secondary_validation_status": null,
+    "secondary_validator_id": null,
+    "reachable_on_submit": true,
+    "raw_value": null,
+    "decimal_separator": null,
+    "validatable": false,
+    "session_ident": null,
+    "lock_version": 1,
+    "validation_color": null,
+    "validation_escalate": false,
+    "validator_validation_date": null,
+    "secondary_validation_date": null,
+    "automatic_validation": false,
+    "similar_value_id": null,
+    "similar_value_count": null
+}
+```
+
+## POST values/{variable} - Indicate the Hyperlink URL
+Post the hyperlink for the evidence document as a string.
+
+```shell
+curl https://api.gresb.com/api/v0/entities/{entity_id}/responses/{response_id}/values/PO_1_A1_EVD_LINK_1 \
```

Comment:
I did some research on this and found some resources which indicate {} is the standart template to express parameters in URI-like descriptors. In Postman specific context environment/collection variables are identified with {{}} as you mentioned and if we use this it would align with our collection. However I am inclined to opting for single curly brackets because I see other API documentation, for example in Swagger or Measurabl they also represent it like this.

Reply:
cool cool - then we go for single curly brackets 👍